OFEP-inline-evaluation.md

```diff
@@ -0,0 +1,37 @@
+## Inline Evaluation of Flag Rules
+
+## State: APPROVED
+
+This OFEP seeks to find a solution for near-zero latency flag evaluation for server-side contexts (e.g. not mobile).
+
+## Background
+
+Today, most flag providers make API calls to backing services to understand if a given flag should return e.g. `true` or `false`. Even if that API call is going to a kubernetes sidecar like the flagD case, it introduces latency which may not be acceptable in high scale use-cases. The flag's rules are often evaluated on another service because determining whether the provided context matches a given set of rules requires computation which may be non-trivial or may take into account data not known by the caller.
```

Comment:
Many tools and vendors already do in-process evaluation using a local ruleset. The ruleset is either periodically updated or subscribes to change events.